### DISCUSSION TITLE
ChatGPT：便利與風險並存的人工智慧助手

### DISCUSSION THREAD

**Discussion by DuxtinTai** [1101-GS4719-C/Python-Community-GS4719-C#1191] [General]
這篇文章我想要來分享chatgpt對我的幫助及隱憂
幫助：
作為一個表達不太流利的人，ChatGPT 是一個寫作輔助工具，為我提供各種語言運用和表達方式的建議，幫助我更好地撰寫文章。例如，在生活中常常需要與教授聯繫，不論是有關課程問題或是專題，寫信時需要格外注意遣詞用字的正式程度。這時，ChatGPT 能夠幫助我更快速地寫出一封禮貌的信件。

隱憂：
首先是過度依賴 ChatGPT 的問題，這同時也是請 ChatGPT 代筆的副作用。因為我原本就不太會寫文章，現在有 ChatGPT 幫忙，缺乏練習的情況可能會更加嚴重。長期下來，可能會變得越來越不會表達自己的想法。第二個問題是資安問題。最近，許多 ChatGPT 相關的插件開始出現，提供更方便的服務，但同時也可能會有潛在的木馬問題，因此使用者必須謹慎。另外，官方插件已經推出，但好像只有付費版本才有此服務。相關連結如下：

連結一 ChatGPT假網頁插件開始流行？冒認知名插件、一裝即失FB控制權 | 香港01
 https://www.hk01.com/article/881004?utm_source=01articlecopy&utm_medium=referral
連結二 ChatGPT Plugin免費官方插件推出！申請安裝+使用功能教學
https://www.nmplus.hk/1120660/?utm_campaign=NM_ContentCopy&utm_source=Web-inventory&utm_medium=Content-Copy_NM

**Comment by Lipunpun** [comment]
我也蠻認同這篇文章的，使用AI工具已經是一種趨勢了，他會讓生活、工作更有效率，在這個過程中難免會有一些缺點，但我認為隨著科技的進步，這是必然的結果。

**Comment by zilinng** [comment]
我也有類似的經驗！Chatgpt剛出來的時候我曾經用它幫我修改英文課的作文作業，當時真的覺得超！級！方！便！不管是文法錯誤或是單字使用不正確它都能完美的幫我修正，甚至如果作文寫太短它也能幫我做擴充，不小心寫的太開心導致過於冗長的話也能請它幫忙精簡，然而因為如此我的英文能力在上完那門英文課後完全沒有進步（當初選課就是因為覺得自己英文太破了想進步），反而退步了，後來思考原因就是因為我過於依賴它的輔助，造成沒有使用的時候寫不出任何東西，真的覺得這樣非常糟糕！

**Comment by KevinCheng42** [comment]
我也覺得Chatgpt是一個很優異的文章撰寫輔助工具，它可以根據你的要求，例如：作為演講稿、信件，或是知性文章，修改用辭和文章語氣。但是，過度依賴Chatgpt會使自己的寫作能力退步，本末倒置，因此，我們可以在有限的範圍內，如修飾文辭，使用Chatgpt輔助，不僅可以讓文章詞藻更華麗，亦可從它修改後的文句中學習～
